examples/js/loaders/OBJLoader2.js

```diff
@@ -221,7 +220,7 @@ THREE.OBJLoader2 = (function () {
 		}
 
 		Parser.prototype.setDebug = function ( debug ) {
-			this.debug = ( debug == null ) ? this.debug : debug;
+			this.debug = Boolean( debug ) ? debug : this.debug;
```

Comment:
I don't like this `Boolean()` stuff...

Consider this case:

```
> Boolean( "true" )
< true
```

I think you should, instead, do:

```js
this.debug = debug === true ? debug : this.debug;
```